src/olcs/core/olimageryprovider.js

```diff
@@ -174,7 +175,7 @@ Object.defineProperties(olcs.core.OLImageryProvider.prototype, {
  */
 olcs.core.OLImageryProvider.prototype.handleSourceChanged_ = function() {
   if (!this.ready_ && this.source_.getState() == 'ready') {
-    const proj = this.source_.getProjection();
+    const proj = olcs.util.getSourceProjection(this.source_);
```

Comment:
The line below can be simplified:
`this.projection_ = olcs.util.getSourceProjection(this.source_) || this.fallbackProj_`.